__tests__/Unit/Components/Tasks/SuggestionBox.test.tsx

```diff
@@ -0,0 +1,50 @@
+import SuggestionBox from '@/components/tasks/SuggestionBox/SuggestionBox';
+import { render, screen, fireEvent } from '@testing-library/react';
+import { DUMMY_PROFILE as placeholderImageURL } from '@/constants/display-sections';
+import { GithubInfo } from '@/interfaces/suggestionBox.type';
+
+const gitInfo: GithubInfo[] = [
+    {
+        github_id: 'fakhruddinkw',
+        profileImageUrl: placeholderImageURL,
+    },
+    {
+        github_id: 'iamitprakash',
+        profileImageUrl: placeholderImageURL,
+    },
+];
+
+describe('Suggestion Box', () => {
+    const onSelectAssignee = jest.fn();
+    it('should render User not found text, if the suggestion prop contains empty array', () => {
+        render(
+            <SuggestionBox
+                suggestions={[]}
+                onSelectAssignee={onSelectAssignee}
+            />
+        );
+
+        const userNotFound = screen.getByTestId('user_not_found');
+        expect(userNotFound).toBeInTheDocument();
+    });
```

Comment:
try using getByRole instead of getByTest
https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#not-querying-by-text

Reply:
Please make changes everywhere in your PR

Reply:
That is opinion of the author.. that doesnt make this enforced. We can choose regardless.

Reply:
That is opinion of author.. we can choose regardless. This is not something which is enforced. @Pratiyushkumar

Reply:
Regardless of the author's choice, there's a reason why the docs mention the importance/ priority of the APIs to be used.  If possible please make the changes.